chapters/synchronous.tex

```diff
@@ -551,7 +551,7 @@ \section{Clock Constructors}\label{clock-constructors}
   \end{example}
 \end{itemize}
 
-\section{Clocked State Variables}\label{clocked-state-variable}
+\section{Clocked State Variables}\label{clocked-state-variables}
```

Comment:
Do we have a convention for this?  Should labels be updated each time a heading is modified?

(My intention was to use a label corresponding to the term being defined.)

Reply:
The labels were generally made to match the section-name in an automated way (so no abbreviations or other changes etc), since that makes it more predictable. One could argue for keeping the previous label here, but that was discrete-states.

Reply:
Yes, I found the old label completely misleading when it neither related to the section title nor the contents of the section anymore, so I updated it.